packages/components/src/components/icon/rebranding/ic-rebranding-deriv-x.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 32 32"><path fill="#FF444F" d="M0 6a6 6 0 0 1 6-6h20a6 6 0 0 1 6 6v20a6 6 0 0 1-6 6H6a6 6 0 0 1-6-6V6z"/><path fill="#E12E3A" d="M6 0a6 6 0 0 0-6 6v.174l29.914-4.721A5.976 5.976 0 0 0 26 0H6zm24.532 2.068L25.044 32H26a6 6 0 0 0 6-6V6c0-1.504-.553-2.88-1.468-3.932z"/><path fill="#fff" d="M9.544 19.066c.103.01.22.019.35.028h.49c1.092 0 1.9-.275 2.422-.826.532-.55.798-1.311.798-2.282 0-1.017-.252-1.787-.756-2.31-.504-.523-1.302-.784-2.394-.784a7.9 7.9 0 0 0-.462.014c-.158 0-.308.01-.448.028v6.132zm6.314-3.08c0 .84-.13 1.573-.392 2.198a4.039 4.039 0 0 1-1.12 1.554c-.476.41-1.06.719-1.75.924-.69.205-1.465.308-2.324.308-.392 0-.85-.019-1.372-.056a10.52 10.52 0 0 1-1.54-.196v-9.45a12.018 12.018 0 0 1 1.568-.182 21.39 21.39 0 0 1 1.414-.056c.83 0 1.582.093 2.254.28.682.187 1.265.48 1.75.882a3.92 3.92 0 0 1 1.12 1.54c.262.625.392 1.377.392 2.254z"/><path fill="#fff" d="M22.021 20.844a25.705 25.705 0 0 0-.896-1.638c-.317-.56-.686-1.13-1.106-1.708-.13.168-.298.401-.504.7-.196.299-.396.616-.602.952-.196.336-.382.658-.56.966-.177.308-.308.55-.392.728H15.47c.411-.756.887-1.55 1.428-2.38a84.637 84.637 0 0 1 1.834-2.646l-3.122-4.676h2.646l1.89 3.024 1.848-3.024h2.492l-3.08 4.704a32.63 32.63 0 0 1 1.974 2.828c.532.859.952 1.582 1.26 2.17h-2.618z"/></svg>
```

Comment:
Why we have 3 different files for the same icon? 
```
ic-rebranding-deriv-x.svg
ic-rebranding-derivx.svg
ic-rebranding-dxtrade-dashboard.svg
```

Reply:
Deriv x icon is used in different platforms with different names. `IcAppstoreDerivx`, `IcAppstoreDerivX`, ... .
That's why we need to have same icons with different names. We need to have a refinement for this matter since we are using duplicate icons but it is outside of scope of this PR.